### DISCUSSION TITLE
What's the point of the file .config/labwc/environment (or the root path)

### DISCUSSION THREAD

**Discussion by ludgie** [labwc/labwc#1291] [Q&A] [answered]
I'm wondering what's the point of creating a environment file under labwc folder if it's only for settings variables that I can set under /etc/profile, .bashrc or /etc/environment (or even under /etc/security/pam_env.conf if you use seatd)

Does this file start more before than the others or is it just a reference of the /etc/environment file?

**Comment by Consolatis** [comment]
Basically its used to have labwc or wayland specific environment variables in there. If you'd use system wide config files for those you would for example also have them set in any random X session. You could in fact also just set them system wide or manually before starting labwc if you prefer it that way.

**Comment by johanmalm** [comment, marked as answer]
Nothing special about it. You can set the env vars in session files or shell-profiles.

**Comment by ludgie** [comment]
Ok, thank you
